contrib/artifactory/src/mill/contrib/ArtifactoryPublishModule.scala

```diff
@@ -0,0 +1,52 @@
+package mill.contrib
```

Comment:
```suggestion
package mill.contrib.artifactory
```

Let's keep the namespace clean.